server/project.go

```diff
@@ -327,10 +327,6 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 	})
 
 	g.POST("/project/:projectID/repository/:repositoryID/sql-review-ci", func(c echo.Context) error {
-		if !api.FeatureEnabled(api.FeatureVCSSQLReviewWorkflow, s.profile.Mode) {
```

Comment:
Do we still need the feature guard check?

Reply:
I think we can remove it now